package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@hyparam/monorepo",
+  "version": "0.0.0",
+  "private": true,
+  "license": "MIT",
+  "workspaces": [
+    "packages/components",
+    "packages/cli",
+    "apps/hightable-demo",
+    "apps/hyparquet-demo"
+  ],
+  "scripts": {
+    "build": "npm run build -ws",
+    "coverage": "npm run coverage -ws",
+    "lint": "npm run lint -ws",
+    "test": "npm test -ws"
+  },
+  "overrides": {
+    "highlight.js": "11.10.0",
+    "hightable": "0.7.2",
+    "hyparquet": "1.6.4",
+    "hyparquet-compressors": "1.0.0",
+    "react": "18.3.1",
+    "react-dom": "18.3.1"
+  },
+  "devDependencies": {
+    "@eslint/js": "^9.16.0",
```

Comment:
`@eslint/js`, `eslint`, and `globals` have non-pinned versions

Reply:
hmmm, right

Reply:
fixed with https://github.com/hyparam/hyperparam-cli/pull/130/commits/8195b21d416deb940884b3f1f191c6552cf951f1